app/src/main/java/org/schabi/newpipe/fragments/detail/VideoDetailFragment.java

```diff
@@ -583,6 +585,9 @@ protected void initViews(final View rootView, final Bundle savedInstanceState) {
     protected void initListeners() {
         super.initListeners();
 
+        videoTitleRoot.setLongClickable(true);
```

Comment:
I don't think this line is needed, as `setOnLongClickListener` already makes the view long clickable: https://developer.android.com/reference/android/view/View#setOnLongClickListener(android.view.View.OnLongClickListener)